openapi/openapi.yaml

```diff
@@ -554,6 +555,10 @@ components:
                 version_id:
                   type: string
                   example: "29"
+            runtime_environment:
+              type: object
```

Comment:
Can it be more restrictive and define all the info present?

Reply:
Yes of course :)